Tests/HummingbirdCompressionTests/CompressionTests.swift

```diff
@@ -140,11 +140,16 @@ class HummingBirdCompressionTests: XCTestCase {
         struct VerifyResponseBodyChunkSize<Context: RequestContext>: RouterMiddleware {
             let bufferSize: Int
 
-            struct Writer: ResponseBodyWriter {
-                var parentWriter: any ResponseBodyWriter
+            final class Writer: ResponseBodyWriter {
```

Comment:
Do we have a strong preference here? Since the type is internal I don't think we necessarily want to change this